clientapi/routing/memberships.go

```diff
@@ -35,6 +36,16 @@ type getJoinedRoomsResponse struct {
 	JoinedRooms []string `json:"joined_rooms"`
 }
 
+// https://matrix.org/docs/spec/client_server/r0.6.0#get-matrix-client-r0-rooms-roomid-joined-members
```

Comment:
gomatrix has this struct already: https://github.com/matrix-org/gomatrix/blob/9e7906b6766dd084ca15530f16ef33812ae30da5/responses.go#L64

Reply:
It makes it a pain to unmarshal the content into due to that form being inlined. Also, I'd like to slowly remove our dependency on gomatrix as it's unmaintained (or otherwise actually maintain it).